packages/happo-core/src/getLastResultSummary.js

```diff
@@ -3,10 +3,12 @@ const path = require('path');
 
 const { config } = require('./config');
 
-module.exports = function getLastResultSummary() {
+function getLastResultSummary() {
   const resultSummaryJSON = fs.readFileSync(
     path.join(config.snapshotsFolder, config.resultSummaryFilename),
     'utf8',
   );
   return JSON.parse(resultSummaryJSON);
-};
+}
+
+module.exports = getLastResultSummary;
```

Comment:
meh, I'd rather we change to `export default` and `import` everywhere but that can certainly wait for another time.